client/printer.go

```diff
@@ -50,42 +55,36 @@ func NewPrinter(config *PrinterConfig) (*Printer, error) {
 
 // SetEnv logs out 'setenv'
 func (p *Printer) SetEnv(map[string]string) error {
-	p.logger.Printf("[INFO] (client.printer) setting workspace environment: "+
-		"'%s', workingdir: '%s'", p.workspace, p.workingDir)
+	p.logger.Info("setting environment for workspace")
 	return nil
 }
 
 // SetStdout logs out 'set standard out'
-func (p *Printer) SetStdout(w io.Writer) {
-	p.logger.Printf("[INFO] (client.printer) setting standard out for workspace: "+
-		"'%s', workingdir: '%s'", p.workspace, p.workingDir)
+func (p *Printer) SetStdout(_ io.Writer) {
```

Comment:
I think omitting the `_` variable would work too, and just leave the type `SetStdout(io.Writer) {}`. Same with the other methods for `Printer`.